VK_App/VK_App/Information/PhotoInformation/PhotoInformation.swift

```diff
@@ -0,0 +1,9 @@
+// PhotoInformation.swift
+// Copyright © RoadMap. All rights reserved.
+
+import RealmSwift
+
+/// Ответ  серера о фотографиях пользователя
```

Comment:
опечатка, другие респонзы (где в названии response) тоже лучше так описать - что это ответ сервера 
```suggestion
/// Ответ сервера о фотографиях пользователя
```